src/viewer/AudioViewer.jsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react'
+import { Icon } from 'cozy-ui/transpiled/react'
 
 import withFileUrl from './withFileUrl'
 import styles from './styles'
+import AudioIcon from './icons/icon-type-audio.svg'
 
 const AudioViewer = ({ file, url }) => (
   <div data-test-id="viewer-audio" className={styles['pho-viewer-audioviewer']}>
+    <Icon icon={AudioIcon} width={160} height={140} />
```

Comment:
Genuine interrogation: since `width` & `height` props expects only numbers what's the main difference between:
```jsx
<Icon icon={AudioIcon} width={160} height={140} />
```
and
```jsx
<Icon icon={AudioIcon} width="160" height="140" />
```

Reply:
I think the doc says number or strings? But otherwise, `160` has a type of `Number` where as `"160"` has a type of `String`. So the 2 will behave differently depending on what you do with them in js. So for example `"160" + 3 = "1603"` whereas `160 + 3 = 163`.

AFAIK we don't actually manipulate these variables though, they simply end up in the HTML markup, so `160` (the number) ends up converted into `"160"` (the string).